### PR TITLE
Update powershell-beginners-guide.md

### PR DESCRIPTION
**Pull request by aavdberg**
clear is also an alias for clear-host.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [x] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: docs/learning-powershell/powershell-beginners-guide.md [hunk 1]
```diff
@@ -96,12 +96,12 @@ Alias           type -> Get-Content
...
 ```
 
-As you can see `cls` is an alias of `Clear-Host`.
+As you can see `cls` or `clear` is an alias of `Clear-Host`.
 
 Now try it:
 ```powershell
 PS /> Get-Process
-PS /> cls
+PS /> cls or clear
```
Comment (iSazonov) [inline docs/learning-powershell/powershell-beginners-guide.md]: It is command line example, we can not execute `cls or clear`.

Comment (anmenaga) [reply docs/learning-powershell/powershell-beginners-guide.md]: Fixed. Comment above and `Get-Alias` output send the message across, so removed ' or clear' from this example.

**Review by anmenaga — APPROVED** [review]

**Comment by anmenaga** [comment]
@iSazonov please take a look again.

**Review by iSazonov — APPROVED** [review]

**Review by adityapatwardhan — APPROVED** [review]

**Comment by iSazonov** [comment]
@aavdberg Thanks for your contribution!

**Pull request merged**